priv/samples/tflite.livemd

```diff
@@ -0,0 +1,143 @@
+# Use TensorFlow Lite on Nerves Livebook
+
+```elixir
+IO.puts("Mix.install/2 is not supported on Nerves Livebook (yet!).")
+```
+
+## Introduction
+
+TensorFlow Lite is a stripped-down version of
+[TensorFlow](https://en.wikipedia.org/wiki/TensorFlow), a free and open-source
+software library for machine learning and artificial intelligence.
+
+In Elixir, we can use TensorFlow Lite through the package
+[`tflite_elixir`](https://github.com/cocoa-xu/tflite_elixir), which does the
+TensorFlow Lite Elixir bindings with optional [Edge
+TPU](https://en.wikipedia.org/wiki/Tensor_Processing_Unit#Edge_TPU) support.
+
+In this notebook, we will perform image classification with pre-trained model.
+
+When working with TensorFlow Lite, we make extensive use of the functions from
+TFLiteElixir, so it's useful to alias the module as something shorter.
+
+```elixir
+alias TFLiteElixir, as: TFLite
+alias TFLiteElixir.Interpreter
+```
+
+## Download model and labels
+
+It is worth mentioning that the only writable partition on a Nerves device, by
+default, is the `/root` partition, which is also symlinked as `/data`.
+
+```elixir
+use Toolshed
+cmd("ls -al /root")
+```
+
+```elixir
+tmp_dir = "/data/tmp"
+File.mkdir_p!(tmp_dir)
+
+download = fn url ->
```

Comment:
Bumblebee has a really nice file downloader that gives progress bars. I'm also not sure where it saved files. I'll try it out in a bit. I'm also curious where it saved them since I never looked. 

Of course, using `req` is certainly fine with me, since it seems silly to add a dependency on Bumblebee just for the file downloader if the Bumblebee downloader is nontrivial. It would be nice to be consistent with download locations, though.